R/FilteredData.R

```diff
@@ -359,6 +377,21 @@ FilteredData <- R6::R6Class( # nolint
         args = c(list(dataset), dataset_args, list(dataname = dataname))
       )
 
+      private$reactive_data[[dataname]] <- reactive({
+        env <- new.env(parent = parent.env(globalenv()))
+        for (idx in seq_along(private$eval_env)) {
+          env[[names(private$eval_env)[idx]]] <- if (is.reactive(private$eval_env[[idx]])) {
+            private$eval_env[[idx]]()
+          } else {
+            private$eval_env[[idx]]
+          }
+        }
```

Comment:
eval_env is empty - there is no call where something is added to this list

Reply:
And this is the key to solve parent->child relationship. You need `set_dataset` method in `CDISCFIlteredData`  and practically copy paste this reactive. In CDISCFilteredData this reactive should have something like

```r
env[[parent_name]] <- private$reactive_data[[parent_name]]
```

Reply:
yup so this seems to do it:
```
env[[paste0(private$parents[[dataname]], "_FILTERED")]] <-
            private$reactive_data[[private$parents[[dataname]]]]()
```